### DISCUSSION TITLE
Deceptive site ahead

### DISCUSSION THREAD

**Discussion by Z-ayat** [community/community#28888] [Actions] [answered]
![image](https://user-images.githubusercontent.com/45783300/183640685-b4f310d5-ae8d-48d3-923f-921a0c3f8f5b.png)
am getting this problem while trying to access any project i did recently after cloning Facebook / Udemy / Khamsat / Netflix Websites 
i deleted all these repository and still have that caution so what am i able to do ?

**Comment by airtower-luna** [comment, marked as answer]
Check the Google Safe Browsing documentation on the matter: https://developers.google.com/search/docs/advanced/security/social-engineering?hl=en#fixing-the-problem

Good luck! :slightly_smiling_face:

**Reply by Z-ayat** [reply]
Thanks for the reply will do my best
